services/client_test.go

```diff
@@ -111,5 +108,6 @@ func (s *ClientTestSuite) TearDownSuite() {
 }
 
 func TestClient_Authenticate(t *testing.T) {
+	t.Skip()
```

Comment:
Why is it skipped?

Reply:
Temporary solution, this will be removed later

Reply:
Please remove it if it is not used